src/cpp/command.cpp

```diff
@@ -137,19 +120,20 @@ void Command::add_field(const char* field, bool is_key)
     keys.
     */
 
-    int field_size = std::strlen(field);
-    char* f = (char*) malloc(sizeof(char)*(field_size));
-    std::memcpy(f, field, sizeof(char)*field_size);
-    this->_local_fields.push_back({f, _fields.size()});
-    this->_fields.push_back(std::string_view(f, field_size));
+    size_t field_size = std::strlen(field);
+    char* f = new char[field_size];
+    if (f == NULL)
```

Comment:
This one too.  I'll probably stop tagging these and we just search for this syntax if we want to chage.